src/main/java/tc/oc/pgm/util/FlairUtils.java

```diff
@@ -0,0 +1,92 @@
+package tc.oc.pgm.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.UUID;
+import javax.annotation.Nullable;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import tc.oc.identity.Identities;
+import tc.oc.named.NameRenderer;
+import tc.oc.pgm.Config;
+import tc.oc.pgm.Config.Flairs.Flair;
+import tc.oc.pgm.api.match.MatchManager;
+import tc.oc.pgm.api.party.Party;
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.tablist.PlayerTabEntry;
+import tc.oc.tablist.TabManager;
+
+public class FlairUtils {
+
+  private static Map<UUID, String> flairCache = new HashMap<UUID, String>();
```

Comment:
Don't use `static` here, also this should probably be a `Cache<UUID, String> cache = CacheBuilder.newBuilder() ...`

Reply:
How would a `CacheBuilder` work when this exists just to store the value given by the API?